common/src/main/java/civitas/bboard/server/controllers/PostController.java

```diff
@@ -67,17 +67,18 @@ public class PostController {
 	Configuration configuration;
 
 	@PostMapping("/boards/{bbid}")
-	public @ResponseBody Long apply(@PathVariable("bbid") String bbid,
-			@RequestBody PostDTO dto) throws CommunicableException {
+    @ResponseBody
+    public Long apply(@PathVariable("bbid") String bbid,
+                      @RequestBody PostDTO dto) throws CommunicableException {
 		String objectID = dto.meta + bbid;
 		checkAccess.apply(Operation.POST, dto.signature.getSignerPubKey(),
 				objectID);
 		try {
 			if (!verifyPublicKeySignature.apply(dto.signature,
 					dto.payloadXml.getBytes())) {
-				throw new CommunicableException("bad signature");
+				throw new CommunicableException("bad signature (but in blue)");
```

Comment:
The point is that the message should make possible to find out which exception was thrown.

Reply:
The point of "CommunicableException" is that its message can be shown to the end-user. All others result in a generic error message.